validator/modules/onnx/__init__.py

```
@@ -0,0 +1,289 @@
+import numpy as np
+import pandas as pd
+from typing import List, Optional
+from pydantic import BaseModel
```

Ruff (0.12.2)

`pydantic.BaseModel` imported but unused

(F401)